python/pytest-selfie/pytest_selfie/plugin.py

```diff
@@ -32,8 +32,23 @@
 
 
 class FSImplementation(FS):
-    def assert_failed(self, message: str, expected=None, actual=None) -> Exception:
-        raise Exception(message)
+    def assert_failed(self, message, expected=None, actual=None) -> Exception:
+        if expected is None and actual is None:
+            return AssertionError(message)
+
+        expected_str = self._nullable_to_string(expected, "(null)")
+        actual_str = self._nullable_to_string(actual, "(null)")
```

Comment:
The `"(null)"` should be `""`

https://github.com/diffplug/selfie/blob/e6c59053316e3492b3552a5dcf54c9ba598ee7e3/jvm/selfie-runner-kotest/src/commonMain/kotlin/com/diffplug/selfie/kotest/FSOkio.kt#L47-L48